### ISSUE TITLE
Use of reduce_max in input

### ISSUE THREAD

**Issue by akshay-iyer** [weichen582/RetinexNet#18]
In the beginning of DeconNet function, there are these two ines:

_input_max = tf.reduce_max(input_im, axis=3, keepdims=True)
input_im = concat([input_max, input_im])_

Could you tell what it does and what is the use of the same?
